ideascube/tests/test_settings.py

```diff
@@ -14,12 +14,6 @@ def test_all_settings(request):
         module = path.purebasename
         module = '.conf.%s' % module
 
-        try:
-            settings = importlib.import_module(module, package="ideascube")
-
-        except Exception as e:
-            print(path)
-            print(module)
-            pytest.fail(str(e))
+        settings = importlib.import_module(setting_module, package="ideascube")
```

Comment:
> Why not try/catch the exception anymore ?

Because if there's an exception, `pytest` considers the test to have failed and displays the exception, which is exactly what we want.

If we catch it and tell `pytest.fail(e)` as we were doing, then we get an output covering several pages of terminal, with lots of unnecessary boilerplate, making finding the actual error much harder.

> There is no setting_module here. On latter commit yes, but not yet.

The joys of rebasing. 😆